### DISCUSSION TITLE
Speed up Lambdify

### DISCUSSION THREAD

**Discussion by Simone2021space** [sympy/sympy#24383] [Q&A] [answered]
Hi,
I am using lambdify function for a matrix 8x1 with several dynamicssymbols to replace, but it is extremely slow.
Are there general rules to speed up it?

**Comment by moorepants** [comment, marked as answer]
You could try the `cse=True` flag, if you haven't. For large expressions, doing cse + code printing can be faster overall than simply code printing large expressions.

**Reply by moorepants** [reply]
Also, what is slow? The `lambdify()` call or the evaluation of the resulting function?

**Reply by Simone2021space** [reply]
Do you have an example of using cse?  
it got stuck at lambdify() call. I don't need to print the code because the matrix expression is huge.
would be numba a good solution?

**Reply by moorepants** [reply]
`lambdify(..., ..., cse=True)`

**Reply by Simone2021space** [reply]
doesn't work for me, I got this error
`TypeError: lambdify() got an unexpected keyword argument 'cse'`

**Reply by moorepants** [reply]
You'll need the lastest version of sympy.
